src/sentry/db/models/manager.py

```diff
@@ -340,6 +340,8 @@ def from_event_id(self, id_or_event_id, project_id):
 
             id_or_event_id = event.event_id
 
+            project_id = event.project_id
```

Comment:
Why do we need to do this?

Reply:
Oh yeah I guess we don't need this. I was kinda thinking we could have this function also handle the case where it's being called with a event ID as database key and no project, but that might be unnecessary

Reply:
Removing

Reply:
Are we actually able to fetch an event without a project id? I think we shard on project, so I'm not sure there's a good way to do so?

Reply:
Yeah we should probably just never call this without a project ID